documentation/slate/source/includes/apis.md

```diff
@@ -260,35 +266,42 @@ paginate results.
 -   [vehicle_journeys](#vehicle-journey)
 -   [disruptions](#disruption)
 
-### Parameters
+### Shared parameters
 
 #### <a name="depth"></a>depth
 
-This tiny parameter can expand Navitia power by making it more wordy.
+You are looking for something, but Navitia doesn't output it in your favorite endpoint?
+You want to request more from navitia feed?
+You are receiving feeds that are too important and too slow with low bandwidth?
+You would like it to be GraphQL?
```

Comment:
Talking about GraphQL should be an issue. If you write about it in the doc, and navitia does not manage it, it should be deceptive, no?